docs/deploy.md

```diff
@@ -37,51 +39,32 @@ NOTE: You also need to install `Lua` 5.1+ if you want to use the Plugin Orchestr
 
 2. Check environment variables
 
-- enable Go MODULE
-
-```sh
-$ go env -w GO111MODULE=on
-```
-
 - For most users in China, we could use [Goproxy](https://goproxy.cn/) to speed up downloading modules.
 
 ```sh
 $ go env -w GOPROXY=https://goproxy.cn,direct
 ```
 
-3. Build
-
-```sh
-$ api/build.sh
-```
-
 The bundled files are located in the root directory `/output`.
 
-## Build the web
+### Web Dependencies
 
 This project is initialized with [Ant Design Pro](https://pro.ant.design). The following are some quick guides for how to use.
 
 1. Make sure you have `Node.js(version 10.0.0+)` installed on your machine.
 2. Install [yarn](https://yarnpkg.com/).
-3. Install dependencies:
-
-```sh
-$ cd /web
-
-$ yarn install
-```
 
-4. Build
+### Build
 
 ```sh
-$ yarn build
+$ make build
 ```
 
-The bundled files are located in the root directory `/output/html`.
+The bundled files are located in the root directory `output`.
```

Comment:
Have you modified the final output folder for the web?

Reply:
no, I do not .
the line under `make build`, contains `api` and `web`, so modify `/output/html` to `output`.